code/api/src/config/config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "development": {
+    "username": "postgres",
+    "password": null,
+    "database": "crate",
+    "host": "127.0.0.1",
+    "dialect": "postgres"
+  },
+  "production": {
+    "username": "postgres",
+    "password": null,
+    "database": "crate",
+    "host": "127.0.0.1",
+    "dialect": "postgres"
+  }
+}
```

Comment:
Still not sure why this was created and why we can't use the db.json. Will research.